DependencyInjection/EzSystemsEzSupportToolsExtension.php

```diff
@@ -23,5 +36,41 @@ public function load(array $configs, ContainerBuilder $container)
         $loader = new Loader\YamlFileLoader($container, new FileLocator(__DIR__ . '/../Resources/config'));
         $loader->load('services.yml');
         $loader->load('default_settings.yml');
+
+        $configuration = $this->getConfiguration($configs, $container);
+        $config = $this->processConfiguration($configuration, $configs);
+
+        if (isset($config['system_info']) && $config['system_info']['powered_by']['enabled']) {
+            $container->setParameter(
+                'ezplatform_support_tools.system_info.powered_by.name',
+                $this->getPoweredByName(
+                    $container,
+                    $config['system_info']['powered_by']['release']
+                )
+            );
+        }
+    }
+
+    private function getPoweredByName(ContainerBuilder $container, ?string $release): string
+    {
+        // Autodetect product name if configured name is null (default)
+        $vendor = $container->getParameter('kernel.root_dir') . '/../vendor/';
+        if (is_dir($vendor . EzSystemInfoCollector::COMMERCE_PACKAGES[0])) {
+            $name = 'eZ Commerce';
+        } elseif (is_dir($vendor . EzSystemInfoCollector::ENTERPISE_PACKAGES[0])) {
+            $name = 'eZ Platform Enterprise';
+        } else {
+            $name = 'eZ Platform';
+        }
```

Comment:
So the old name still applies to 2.5 then, right? Just double checking.

Reply:
yes, so this, and maybe others parts needs adjustments when we merge up.

For 3.2+ I was also thinking we should have Platform/Core for OSS and just DXP for all flavors of Enterprise to reflect the new naming.

Reply:
MERGE UP CHANGES:
- Product Name
- Version number from constant